app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -176,4 +178,32 @@ def parse_image(image, is_public)
 
     return uid, new_result
   end
+
+  def parse_vapp_template(vapp_template)
+    uid = vapp_template.id
+
+    begin
+      # The content of the template is the OVF specification of the vApp template
+      content = @connection.get_vapp_template_ovf_descriptor(uid).body
+
+      new_result = {
+        :type        => ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate.name,
+        :ems_ref     => uid,
+        :name        => vapp_template.name,
+        :description => vapp_template.description,
+        :orderable   => false,
+        :content     => content,
+        # By default #save_orchestration_templates_inventory does not set the EMS
+        # ID because templates are not EMS specific. We are setting the EMS
+        # explicitly here, because vapps are specific to concrete EMS.
+        :ems_id      => @ems.id
```

Comment:
As the above comment implies, we have to store the id of the EMS as otherwise the vApp template is useless (for now). Eventually, we may need to support deployment of a vApp into EMS A from a vApp template from EMS B. This would involve creation of a vApp template in EMS A (based on downloaded OVF) and deployment of that vApp.